packages/react-sdk/src/components/elements/image/ImageDisplay.tsx

```diff
@@ -46,14 +46,22 @@ const Image = styled('image', {
   visibility: loading ? 'hidden' : 'visible',
 }));
 
+function isSVG(svg: string): boolean {
+  const parser = new DOMParser();
+  const svgDom = parser.parseFromString(svg, 'image/svg+xml');
```

Comment:
What happens, if we try to parse a PNG here?

Should this be more efficient, for example by testing if the first N bytes container `<svg` or something like that?

Also similar is done in other places. Please extract into a utility module.